ubuntu/gpu/base/Dockerfile

```diff
@@ -1,57 +1,20 @@
-FROM databricksruntime/minimal:latest
+# See https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/ubuntu18.04/10.1/runtime/cudnn7/Dockerfile
+FROM nvidia/cuda:10.1-cudnn7-runtime-ubuntu18.04
```

Comment:
What is the motivation for switching the base image?

Reply:
The NVIDIA dependencies are tricky to manage. For example, cuBLAS 10.2.2 is not compatible with CUDA 10.1 while cuBLAS 10.2.1 is compatible ... it is very easy to make mistakes. So the safest approach is to use NVIDIA CUDA docker directly.